firmware/boron-ins-fsm/src/stateMachine.cpp

```diff
@@ -8,6 +8,7 @@
 #include "im21door.h"
 #include "ins3331.h"
 #include "flashAddresses.h"
+#include "debugFlags.h"
```

Comment:
Since you've included this here, should `bool stateMachineDebugFlag = false;` be removed from this file and maybe put it its own `.cpp` file? Or is that unnecessary?

Reply:
I feel like given the naming of the variable and the structure of our includes, `stateMachine.cpp` feels like the parent of the sensors. Therefore, initializing it there should be fine (unless we move it to the `.ino`, but it doesn't get used there).

Reply:
Sounds good to me. Thanks!